.github/workflows/tag_latest_image.yaml

```diff
@@ -0,0 +1,56 @@
+---
+name: Tag Latest Image
+on:
+  pull_request:
+    branches:
+    - main
+    - master
+    - develop
+    types:
+    - closed
+
+jobs:
+  docker_tag:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out GitHub Repo
+      with:
+        ref: "${{ github.event.pull_request.head.sha }}"
+      uses: actions/checkout@v2
+    -
+      name: Log in to the Container registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.GHCR_USERNAME }}
+        password: ${{ secrets.GHCR_TOKEN }}
+    -
+      name: Set up environment
+      run: |
+        if [ ${{ github.base_ref }} == 'develop' ]; then
+            branch="-""${{ github.base_ref }}"
+        else
+            branch=""
+        fi
+        narrative_version=`grep '\"version\":' src/config.json.templ | awk '{print $2}' | sed 's/"//g'`
+        echo "APP_IMAGE_TAG=ghcr.io/${{ github.repository }}${branch}:pr-${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+        echo "APP_VERSION_IMAGE_TAG=ghcr.io/${{ github.repository }}${branch}_version:pr-${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+        echo "BRANCH=${branch}" >> $GITHUB_ENV
+        echo "VERSION=${narrative_version}" >> $GITHUB_ENV
+
+    - name: Tag and Push to Packages
+      if: github.event.pull_request.draft == false
+      run: |
+        docker pull ${{ env.APP_IMAGE_TAG }}
+        docker tag ${{ env.APP_IMAGE_TAG }} ghcr.io/${{ github.repository }}${{ env.BRANCH }}:latest
+        docker tag ${{ env.APP_IMAGE_TAG }} ghcr.io/${{ github.repository }}${{ env.BRANCH }}:${{ env.VERSION }}
+        docker push ghcr.io/${{ github.repository }}${{ env.BRANCH }}:latest
+        docker push ghcr.io/${{ github.repository }}${{ env.BRANCH }}:${{ env.VERSION }}
+
+        docker pull ${{ env.APP_VERSION_IMAGE_TAG }}
+        docker tag ${{ env.APP_VERSION_IMAGE_TAG }} ghcr.io/${{ github.repository }}${{ env.BRANCH }}_version:latest
+        docker tag ${{ env.APP_VERSION_IMAGE_TAG }} ghcr.io/${{ github.repository }}${{ env.BRANCH }}_version:${{ env.VERSION }}
+        docker push ghcr.io/${{ github.repository }}${{ env.BRANCH }}_version:latest
+        docker push ghcr.io/${{ github.repository }}${{ env.BRANCH }}_version:${{ env.VERSION }}
+        docker images
```

Comment:
Results of running this workflow on the truss branch, with the condition
`if: github.event.pull_request.merged == true`
commented out, and the `docker push` commands just echoed, instead of being run:

<img width="652" alt="tag-app-image" src="https://user-images.githubusercontent.com/556057/162048718-26a45636-59c4-4344-8166-ceaf97b35689.png">
^^ tagged narrative docker image

<img width="751" alt="tag-version-image" src="https://user-images.githubusercontent.com/556057/162048744-4fd138a1-582d-4cbc-a7a4-4851f132fc1a.png">
^^ tagged narrative_version image, followed by the output from `docker images`

Reply:
Results of editing this workflow to run on the truss branch when the pull request is reopened and the condition `if: github.event.pull_request.merged == true` reinstated:

<img width="777" alt="skip-workflow" src="https://user-images.githubusercontent.com/556057/162049603-1a434b58-49f3-42b5-97ac-d4ff6c0539cd.png">